docs/src/test-api/class-test.md

```diff
@@ -469,7 +469,18 @@ Learn more about the execution modes [here](../test-parallel.md).
   test('runs in parallel 2', async ({ page }) => {});
   ```
 
-* Running tests serially, retrying from the start.
+* Running tests in order, retry is per failed test. If one test fails, subsequent tests are still runned.
```

Comment:
I'd not mention the "continue on failure" behavior - that's expected. It is only important to mention that for serial mode, and avoid unnecessary details for the default mode.

```suggestion
* Running tests in order, retrying each failed test independetly.
```